tasks/drop.lasso.yml

```
@@ -0,0 +1,37 @@
+---
+- name: "drop.lasso: Install periodic script"
+  ansible.builtin.template:
+    dest: /usr/local/etc/periodic/daily/900.drop-lasso
+    src: "{{ pf_drop_lasso_template }}"
+    owner: root
+    group: wheel
+    mode: "0755"
+    # keeping a backup will result in the script being run multiple times
+    backup: no
+  when: pf_drop_lasso_enable | bool
+  
+- name: "drop.lasso: Enable periodic script"
+  sysrc:
```

Let's use FQCN. Otherwise, *ansible-lint* complains `syntax-check[unknown-module]`
```suggestion
  community.general.sysrc:
```